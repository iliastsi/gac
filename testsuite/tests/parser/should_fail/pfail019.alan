hello() : proc
  foo : int;
{
  foo = *3
  foo = 3;
}
