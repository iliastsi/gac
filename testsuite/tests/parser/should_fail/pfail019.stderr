pfail019.alan:4:9: Error: Parse error on input `*'
	Not a valid expression

